src/bentoml_cli/cloud.py

```diff
@@ -31,7 +43,8 @@ def cloud_command():
 @cog.optgroup.option(
     "--endpoint",
     type=click.STRING,
-    help="BentoCloud or Yatai endpoint, i.e: https://cloud.bentoml.com",
+    help="BentoCloud or Yatai endpoint, default as https://cloud.bentoml.com",
+    default="https://cloud.bentoml.com",
```

Comment:
Make it default from env var `BENTO_CLOUD_API_ENDPOINT`

Same as `BENTO_CLOUD_API_KEY`